config/clients/go/CHANGELOG.md.mustache

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.3.0
+
+### [0.3.0](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.2.3...v0.3.0) (2023-11-02)
+
+- feat!: initial support for conditions
+
```

Comment:
do we want to call out that ClientWriteRequest's delete is changed as it is a breaking change?